lib/rubocop/cop/lint/no_return_in_begin_end_blocks.rb

```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # Checks for the presence of a *return* inside a *begin..end* block
+      # in assignment contexts.
+      # In this situation the, `return` will take precedence over any
```

Comment:
How about:

_In this situation, the `return` will result in an exit from the current method, possibly leading to unexpected behavior._

I guess that putting a `return` inside a `begin`..`end` is something that a Java or JavaScript programmer might do out of habit, since `return` in those languages just returns from the lambda/block/closure.

Reply:
Yes, I agree. I think your suggestion is nice. Apart from being more concise it's also easier to understand.